code/modules/admin/topic.dm

```diff
@@ -989,9 +989,9 @@
 			if(!check_if_greater_rights_than(M.client))
 				usr << "<span class='danger'>Error: They have more rights than you do.</span>"
 				return
-			M << "<span class='danger'>You have been kicked from the server by [usr.client.ckey].</span>"
-			log_admin("[key_name(usr)] booted [key_name(M)].")
-			message_admins("<span class='adminnotice'>[key_name_admin(usr)] booted [key_name_admin(M)].</span>")
+			M << "<span class='danger'>You have been kicked from the server by [usr.client.holder.fakekey ? "an Administrator" : "[usr.client.ckey]"].</span>"

```

Comment:
you didn't need to embed the real key, as it's already in the string, so you're doing a needless embedding operation in both of these changed lines, when byond is notoriously bad with strings.